admin/templates/Includes/ModelAdmin_ImportSpec.ss

```diff
@@ -1,8 +1,8 @@
 <div class="importSpec" id="SpecFor{$ModelName}">
-	<a href="#SpecDetailsFor{$ModelName}" class="detailsLink"><% sprintf(_t('ModelAdmin_ImportSpec_ss.IMPORTSPECLINK', 'Show Specification for %s'),$ModelName) %></a>
+	<a href="#SpecDetailsFor{$ModelName}" class="detailsLink"><%t ModelAdmin_ImportSpec_ss.IMPORTSPECLINK 'Show Specification for %s' s=$ModelName %></a>
 	<div class="details" id="SpecDetailsFor{$ModelName}">
-	<h4><% sprintf(_t('ModelAdmin_ImportSpec_ss.IMPORTSPECTITLE', 'Specification for %s'),$ModelName) %></h4>
-		<h5><% _t('ModelAdmin_ImportSpec_ss.IMPORTSPECFIELDS', 'Database columns') %></h5>
+	<h4><%t ModelAdmin_ImportSpec_ss.IMPORTSPECTITLE 'Specification for %s' s=$ModelName %></h4>
```

Comment:
I’ve named the injection argument `s`, because it sort of matches `%s`. It doesn’t actually matter what the name is, `i18n::_t()` will replace them in order, just as long as it’s `name=$Value` (it won’t just accept `$Value` on its own).

Really, we should update the use of `%s` in _all_ entities to actual placeholders like `{modelname}`, but that’s a bigger change than this PR.